cognite/powerops/custom_modules/power_model_v1/data_models/containers/Scenario.container.yaml

```diff
@@ -54,73 +54,6 @@ properties:
     autoIncrement: false
     name: source
     description: The source of the model
-  shopStartSpecification:
-    type:
-      list: false
-      collation: ucs_basic
-      type: text
-    nullable: true
-    autoIncrement: false
-    name: shopStartSpecification
-    description: The specification for when to start the shop
-  shopEndSpecification:
-    type:
-      list: false
-      collation: ucs_basic
-      type: text
-    nullable: true
-    autoIncrement: false
-    name: shopEndSpecification
-    description: The specification for when to end the shop
-  shopStart:
-    type:
-      list: false
-      type: date
-    nullable: true
-    autoIncrement: false
-    name: shopStart
-    description: The time to start the shop
-  shopEnd:
-    type:
-      list: false
-      type: date
-    nullable: true
-    autoIncrement: false
-    name: shopEnd
-    description: The time to end the shop
-  shopStartTimestamp:
-    type:
-      list: false
-      type: timestamp
-    nullable: true
-    autoIncrement: false
-    name: shopStart
-    description: The time to start the shop
-  shopEndTimestamp:
-    type:
-      list: false
-      type: timestamp
-    nullable: true
-    autoIncrement: false
-    name: shopEnd
-    description: The time to end the shop
-  bidDate:
-    type:
-      list: false
-      type: date
-    nullable: true
-    autoIncrement: false
-    name: bidDate
-    description: The date of the bid
-  shopVersion:
-    type:
-      list: false
-      collation: ucs_basic
-      type: text
-    nullable: true
-    autoIncrement: false
-    name: shopVersion
-    description: The version of SHOP to run
   isReady:
```

Comment:
Didn't we agree to remove isReady (or what do we need it for)?

Reply:
And we should remove `modelFile` too - we find that through the `modelTemplate`.

Reply:
Not sure if `extraFiles` are needed either - what is that for?